quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/integration/web/QuarkusRequestFilter.java

```diff
@@ -76,18 +76,29 @@ private boolean ignoreContext(RoutingContext context) {
 
     private Runnable createBlockingHandler(RoutingContext context) {
```

Comment:
I'm completely lost in the number of filters/interceptors/etc, so got no clue what's going on here and can't review this stuff properly